draft-ietf-rtcweb-jsep.xml

```diff
@@ -2089,9 +2089,16 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               direction:
               <list style="symbols">
 
-                <t>An "a=msid" line, as specified in
-                <xref target="I-D.ietf-mmusic-msid"></xref>, Section
-                2.</t>
+                <t>For each MediaStream that was associated with the
+                transceiver when it was created via addTrack or addTransceiver,
+                an "a=msid" line, as specified in
+                <xref target="I-D.ietf-mmusic-msid"></xref>, Section 2. If a
+                MediaStreamTrack is attached to the transceiver's RtpSender,
+                the "a=msid" lines MUST use that track's ID. If no
+                MediaStreamTrack is attached, a valid ID MUST be generated
+                instead. To avoid fingerprinting, it is RECOMMENDED to generate
+                a UUID <xref target="RFC4122"></xref> using the form in section
```

Comment:
I think this may have come up previously, but do we need to specify a UUID here? Could this just be any valid locally unique id?

That is, could we just say, generate an ID as the implementation would do for a new track?